vue/components/molecules/messages-alert/messages-alert.vue

```diff
@@ -0,0 +1,154 @@
+<template>
+    <transition
+        name="slide-transition"
+        v-on:before-enter="onSlideTransitionBeforeEnter"
+        v-on:after-leave="onSlideTransitionAfterLeave"
+    >
+        <div class="messages-alert" v-show="visible">
+            <div class="messages">
+                <div class="message" v-for="(message, index) in visibleMessages" v-bind:key="index">
+                    <div class="name">
+                        {{ `${message.name}:` }}
+                    </div>
+                    <div class="value">
+                        {{ message.value }}
+                    </div>
+                </div>
+            </div>
+            <div class="buttons">
+                <div class="undo" v-on:click="undo">
+                    {{ "ripe_commons.messages_alert.undo_button" | locale }}
+                </div>
+                <div class="close" v-on:click="close">
+                    {{ "ripe_commons.messages_alert.close_button" | locale }}
+                </div>
+            </div>
+        </div>
+    </transition>
+</template>
+
+<style scoped>
+.messages-alert {

```

Comment:
Reuse from `restrictions-alert`, the only difference should be some new style for `messages`.